ldms/man/ldmsd_sampler_advertisement.man

```diff
@@ -32,7 +32,7 @@ ldmsd_sampler_advertisement - Manual for LDMSD Sampler Advertisement
 
 .IP \fBprdcr_listen_add
 .RI "name=" NAME "
-.RI "[disabled_start=" TURE|FALSE "] [regex=" REGEX "] [ip=" CIDR "] [rail=" SIZE "] [credits=" BYTES "] [rx_rate=" RATE_LIMIT "]"
+.RI "[disable_start=" TURE|FALSE "] [regex=" REGEX "] [ip=" CIDR "]"
```

Comment:
TURE --> TRUE
Do we want the spaces around the values? A a user that copied that would get an error; and likely a very confusing one.